tests/commands/test_commands.py

```diff
@@ -31,8 +32,8 @@ def test_start_raises_if_command_not_implemented():
 
     with pytest.raises(NotImplementedError):
         # Will raise because `scope`, the third parameter,
-        # doesn't contain the function `run_configure`
-        utils.start(parser, ['configure'], {})
+        # doesn't contain the function `run_start`
+        utils.start(parser, ['start'], {})
```

Comment:
Its not a typo. Its just that configure now requires a positional argument and since you can use any command for this test I changed it to `start` to keep it simple.
Its mostly aesthetics.